### PR TITLE
DO NOT MERGE!! - Kyma 1.16 Istio policy migration fixes - DO NOT MERGE!!

### PR DESCRIPTION
**Pull request by nachtmaar**
**Description**

This PR should not be merge into the release branch. Instead a new PR will be created which will contain these changes.
Changes proposed in this pull request:

- Kyma 1.16 Istio policy migration fixes

### CODE REVIEW

File: resources/event-sources/post-upgrade.sh [hunk 1]
```diff
@@ -24,4 +24,10 @@ function delete_orphaned_ksvcs() {
   done
 }
 
+
+function delete_orphaned_policies() {
+    kubectl delete policy -n kyma-integration -l application-name
```
Comment (sayanh) [inline resources/event-sources/post-upgrade.sh:29]: I think we'd need `--ignore-not-found` to make it robust

Comment (nachtmaar) [reply resources/event-sources/post-upgrade.sh:29]:
```suggestion
    kubectl delete policy --ignore-not-found -n kyma-integration -l application-name
```

**Pull request closed without merging**